plugins/woocommerce-admin/client/marketing/data-multichannel/resolvers.ts

```diff
@@ -47,3 +55,48 @@ export function* getRecommendedChannels() {
 		throw error;
 	}
 }
+
+const getTotalFromResponse = ( response: Response ) => {
+	return (
+		parseInt( response.headers.get( 'x-wp-total' ) || '0', 10 ) || undefined
```

Comment:
❓  So if response.headers.get( 'x-wp-total' ) === 0, then return is `undefined`?